src/Feature/Layout/Layout.re

```diff
@@ -26,35 +28,419 @@ let empty = vsplit([]);
 
 let singleton = id => vsplit([window(id)]);
 
+// INTERNAL
+open {
```

Comment:
What does this `open {` do?

Reply:
Learned this trick from @Et7f3 not long ago. It defines an empty module and then opens it, effectively defining top-level functions that won't be exported. Roughly equivalent to:

```
module Internal = {
  ...
};
open Internal;
```

The reason I'm trying this out instead of using an explicit `Internal` module is that you sometimes have an internal function depend on an exported function which in turn depends on an internal function, meaning you need several separate blocks of internal code, which is a bit awkward with explicit `Internal` modules.

What do you think about it?